02.calendar/calendar.js

```diff
@@ -0,0 +1,36 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+const year = parseInt(argv.y) || today.year;
+const month = parseInt(argv.m) || today.month;
+
+const first_date_of_the_month = DateTime.fromObject({
+  year: year,
+  month: month,
+});
+const first_day_of_the_month = first_date_of_the_month.weekday;
+
+/*calendarを表示*/
```

Comment:
このコメントも不要なのでは？
あと前回もコメントしましたが、

```suggestion
// calendarを表示
```

のように `//` を使うことが多いです。

`/* */`を使うにしても

```suggestion
/* calendarを表示 */
```

というようにスペースは挟んだ方が良いです

Reply:
こちらも不要なコメントなので、削除しました。
[🐛 fix: コメントを全削除。](https://github.com/reckyy/js-practices/pull/2/commits/b733335554ccf99c2d4ec77a7fe0c7a44f81dd30)